### PR TITLE
Added GitBlade GUI client

### PR DESCRIPTION
**Pull request by ltomov**

### CODE REVIEW

File: resources/guis.yml [hunk 1]
```diff
@@ -224,3 +224,11 @@ guis:
   price: Free
   license: GNU GPL
   order: 25
+- name: GitBlade
+  url: https://gitblade.com
+  image_tag: guis/gitblade@2x.png
+  platforms:
+  - Linux, Mac, Windows
+  price: 14.99€
```
Comment (pedrorijo91) [inline resources/guis.yml]: on the website it says  6.99€

File: resources/guis.yml [hunk 2]
```diff
@@ -224,3 +224,11 @@ guis:
   price: Free
   license: GNU GPL
   order: 25
+- name: GitBlade
+  url: https://gitblade.com
+  image_tag: guis/gitblade@2x.png
+  platforms:
+  - Linux, Mac, Windows
```
Comment (pedrorijo91) [inline resources/guis.yml]: see L197 (GitAhead) how to define multiple platforms

**Comment by ltomov** [comment]
The requested changes have been done.

File: resources/guis.yml [hunk 3]
```diff
@@ -224,3 +224,13 @@ guis:
   price: Free
   license: GNU GPL
   order: 25
+- name: GitBlade
+  url: https://gitblade.com
+  image_tag: guis/gitblade@2x.png
+  platforms:
+    - Linux
```
Comment (pedrorijo91) [inline resources/guis.yml]: thanks for fixing this :)

I don't want to sound picky, but I just saw on the website that currently only Linux is supported (Mac and Windows coming soon™). In order to keep this list useful, I would prefer to add only Linux support now, and the remaining later, when supported. Would you be ok with that?

Comment (ltomov) [reply resources/guis.yml]: Sure, totally makes sense, I'll fix it.

Comment (ltomov) [reply resources/guis.yml]: Ok, this is fixed.

**Pull request merged**